SPECS/jemalloc/jemalloc.spec

```diff
@@ -0,0 +1,287 @@
+%define      debug_package %{nil}
+Summary:        A General Purpose Malloc Implementation
+Name:           jemalloc
+Version:        5.2.1
+Release:        1%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
+URL:            http://www.canonware.com/jemalloc/
+Source0:        https://github.com/jemalloc/%{name}/releases/download/%{version}/%{name}-%{version}.tar.bz2
+
+%description
+General-purpose scalable concurrent malloc(3) implementation that emphasizes
+fragmentation avoidance and scalable concurrency support
+
+%package devel
+Summary:        Development files for %{name}
+Group:          Development/Libraries
+Requires:       %{name} = %{version}-%{release}
+
+%description devel
+The %{name}-devel package contains libraries and header files for
+developing applications that use %{name}.
+
+%prep
+%setup -q
+
+%build
+%configure
+make %{?_smp_mflags}
+
+%install
+make install DESTDIR=%{buildroot}
+rm %{buildroot}%{_docdir}/%{name}/jemalloc.html
+
+find %{buildroot}%{_libdir}/ -name '*.a' -exec rm -vf {} ';'
+
+%clean
+rm -rf %{buildroot}
+
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
+
+%files
+%{_libdir}/libjemalloc.so.*
+%{_bindir}/jemalloc.sh
+%license COPYING
+%doc README VERSION
+%doc doc/jemalloc.html
+
+%files devel
+%{_includedir}/jemalloc
+%{_bindir}/jemalloc-config
+%{_libdir}/pkgconfig/jemalloc.pc
+%{_bindir}/jeprof
+%{_libdir}/libjemalloc.so
+%{_mandir}/man3/jemalloc.3*
+
+%changelog
+* Mon Dec 14 2020 Henry Li <lihl@microsoft.com> - 5.2.1-1
+- Initial import from Fedora. License Verified.
```

Comment:
Please follow our [spec guidelines](https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/125/Introduction-to-SPEC-Files) in this and all new specs. In this case the proper phrasing should be `Initial CBL-Mariner import from Fedora 29 (license: MIT).`

Also: why Fedora 29? Seems rather old.

Reply:
Resolved